work/templates/work/basic_form.html

```diff
@@ -22,21 +30,23 @@
                 {% include "core/component/form_descriptor.html" with descriptor=form_descriptor %}
 
                 <label>Original catalogue:</label>
-                {{ catalogue_form.catalogue }}
+                <div class="catalogue-div">
+                    {% include "core/component/selectfilter.html" with select_field=catalogue_form.catalogue  datalist_field=catalogue_form.catalogue_list %}
+                </div>
 
                 <div class="row-no-margin">
-                    <button type="submit" name="__goto" value="corr">Correspondents</button>
-                    <button type="submit" name="__goto" value="dates">Dates</button>
-                    <button type="submit" name="__goto" value="places">Places</button>
+                    {% include "work/component/tab_link.html" with name='Correspondents' vname='work:corr_form' iwork_id=iwork_id %}
```

Comment:
these changes will remove all styling from the nav menu:
![image](https://user-images.githubusercontent.com/5488106/218441926-ed63be38-4182-4b12-a840-ea1a8dda502b.png)

Reply:
is css outdated, run 
```
rm $EMLO_CODE_HOME/core/static/core/scss/main.css
```
and too this problem still exist